### ISSUE TITLE
Can we visualize the intermediate result of Query Selection？

### ISSUE THREAD

**Issue by TsingWei** [IDEACVR/DINO#83]
I think it is fun to have the visualization on the top-k boxes selected from the Encoder's output.

**Comment by FengLi-ust** [comment]
Yes, you can just print the boxes of the selected boxes in query selection and visualize them. The quality of boxes after query selection is actually quite good.

**Comment by Andreyao** [comment]
Thank you very much.  And how can we take visualization?  where is the visualized code?
